### PR TITLE
[PR] [WIP] Docs: Started writing release notes.

### PR DESCRIPTION
**Pull request by le-cds**
This pull request starts work on the release notes for 0.7.0. It's a draft as long as we keep fixing issues. However, please check whether I've missed anything or if you can make anything clearer or more informative.

### CODE REVIEW

File: docs/content/downloads/releasenotes/release-0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* [#593](https://github.com/eclipse/elk/pull/593): _ELK Rectangle Packing_ now supports a configurable target width that the algorithm will try to achieve.
+* [#344](https://github.com/eclipse/elk/issues/344): ELK's JSON support nur supports a more relaxed JSON style.
+
+
+### Changes
+
+* [#500](https://github.com/eclipse/elk/issues/500): The Xtext version ELK uses was upgraded to 2.20.
+* [#626](https://github.com/eclipse/elk/issues/626), [#634](https://github.com/eclipse/elk/pull/634): We changed the way port label placement is configured. Previously, there was a choice between inside and outside port labels, with other details configured in other options. We have now made the option an `EnumSet` to move port label options out of the `SizeOptions` enumeration and the `nextToPortIfPossible` option.
+* [#646](https://github.com/eclipse/elk/issues/646), [#647](https://github.com/eclipse/elk/pull/647): _ELK Layered_'s layout options `layering.layerID` and `crossingMinimization.positionID` were renamed to `layering.layerId` and `crossingMinimization.positionId`, respectively. This also impacts the associated layout option constants.
```
Comment (uruuru) [inline docs/content/downloads/releasenotes/release-0.7.0.md:37]: I think they haven't been part of a previous release, have they?

File: docs/content/downloads/releasenotes/release-0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Here's a list of the most noteworthy changes. Head over to GitHub for [the full list](https://github.com/eclipse/elk/milestone/12?closed=1).
+
+
+### New Features and Enhancements
+
+* [#362](https://github.com/eclipse/elk/issues/362), [#613](https://github.com/eclipse/elk/pull/613): Most importantly, we finally have a new logo which appears on all of our websites. Joy!
+* [#105](https://github.com/eclipse/elk/issues/105), [#598](https://github.com/eclipse/elk/pull/598): Configuring spacings properly is one of the most complex things to get right. We thus added [a utility class](https://github.com/eclipse/elk/blob/master/plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/ElkSpacings.java) to help ease the pain a little.
```
Comment (uruuru) [inline docs/content/downloads/releasenotes/release-0.7.0.md:27]: Note to self: add `baseValue` here.

File: docs/content/downloads/releasenotes/release-0.7.0.md (hunk 1 above)
Comment (le-cds) [reply docs/content/downloads/releasenotes/release-0.7.0.md:37]: Oh, good point. @soerendomroes you know best: @uruuru is correct, isn't he?

Comment (soerendomroes) [reply docs/content/downloads/releasenotes/release-0.7.0.md:37]: They are included in elk 6.1(I checked), but I guess no one consumed them yet (despite the KLighD keith branch).

Comment (le-cds) [reply docs/content/downloads/releasenotes/release-0.7.0.md:37]: OK, then I'll leave them here. Thanks for checking. 🙂

File: docs/content/downloads/releasenotes/release-0.7.0.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: "0.7.0"
+menu:
+  main:
+    parent: "ReleaseNotes"
+    weight: 70
+---
+
+* [Release log](https://projects.eclipse.org/projects/modeling.elk/releases/0.7.0)
+* [Documentation](https://download.eclipse.org/elk/updates/releases/0.6.1/elk-0.7.0-docs.zip)
```
Comment (soerendomroes) [inline docs/content/downloads/releasenotes/release-0.7.0.md:10]: I guess this should be https://download.eclipse.org/elk/updates/releases/0.7.0/elk-0.7.0-docs.zip

File: docs/content/downloads/releasenotes/release-0.7.0.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+### New Features and Enhancements
+
+* [#362](https://github.com/eclipse/elk/issues/362), [#613](https://github.com/eclipse/elk/pull/613): Most importantly, we finally have a new logo which appears on all of our websites. Joy!
+* [#105](https://github.com/eclipse/elk/issues/105), [#598](https://github.com/eclipse/elk/pull/598): Configuring spacings properly is one of the most complex things to get right. We thus added [a utility class](https://github.com/eclipse/elk/blob/master/plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/ElkSpacings.java) to help ease the pain a little.
+* [#533](https://github.com/eclipse/elk/issues/533), [#535](https://github.com/eclipse/elk/pull/535): _ELK Box_ and _ELK Rectangle Packing_ now support [content alignment](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-contentalignment.html).
+* [#593](https://github.com/eclipse/elk/pull/593): _ELK Rectangle Packing_ now supports a configurable target width that the algorithm will try to achieve.
+* [#344](https://github.com/eclipse/elk/issues/344): ELK's JSON support nur supports a more relaxed JSON style.
+
```
Comment (soerendomroes) [inline docs/content/downloads/releasenotes/release-0.7.0.md:31]: Maybe we should also add `considerModelOrder` for the `layered` algorithm here.

Comment (le-cds) [reply docs/content/downloads/releasenotes/release-0.7.0.md:31]: Please do, you developed that feature. 🙂

**Pull request merged**